.travis.yml

```diff
@@ -7,7 +7,7 @@ go:
   - 1.9
   - '1.10'
 
-go_import_path: github.com/coreos/dex
```

Comment:
In hindsight, this should let us forgo the package rename. We might be able to split out removing the bill of materials.

Reply:
Technically, yeah... but if someone wanted to depend on post-move dex, those names would have to match, wouldn't they? (Ignoring the new go modules stuff for a moment.)

Reply:
I would feel really bad if someone was importing dex 😬 we don't maintain these packages for general use.

Reply:
😄 It's quite convenient to have an all-in-memory-dex in tests. We do it in a few places... 🤔

Reply:
I suppose the same holds for using its grpc client definitions.